### DISCUSSION TITLE
Classnames don't show up in DOM for only a specific div

### DISCUSSION THREAD

**Discussion by clearly-outsane** [tailwindlabs/tailwindcss#9369] [Help]
I'm not sure how to make this reproduceable or why it's even happening. But this one specific div doesn't get tailwind classes. 
![image](https://user-images.githubusercontent.com/44093134/191099440-f762e495-c741-47c0-9089-a71f0c8f6bed.png)
![image](https://user-images.githubusercontent.com/44093134/191099459-4cb734b2-fc4a-40c8-8fdc-50c351c27cc6.png)

As you can see in the code that div has classnames, but in the DOM, it has no class. If I edit the classname and save, it shows up on hot reload - but once I refresh the page it disappears again. I'm using TailwindCSS with Nextjs. Everything else on the page renders as intended, it's only this random div. 

Could someone direct me as to how I can beginning finding a way to reproduce this? So that if it's a bug, it can be fixed

**Comment by wongjn** [comment]
Tailwind does not directly affect the rendering of your markup. If I could kindly suggest to try asking in Next.js-related circles, you may be able to increases your chances of assistance with your issue.

**Reply by clearly-outsane** [reply]
I'll do that!
